### PR TITLE
19-November-2024 meeting agenda draft

### PR DESCRIPTION
**Pull request by PLeVasseur**
Hey folks :wave: 

Welcome any agenda items we'd like to bring here to go ahead and leave as comments.

### CODE REVIEW

**Comment by ah801** [comment]
"2. Shift meeting time 2 hours earlier" - I'm ok with moving it earlier (or later)
Probably better for the rest of the European participants as well if it's earlier.

File: subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Coding Guidelines Subcommittee Meeting on 2024-11-19
+
```
Comment (JoelMarcey) [inline subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md:2]: Might be good to put the UTC time of the meeting in the agenda too.

**Review by JoelMarcey — APPROVED** [review]
Approved when you are ready to merge.

File: subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md (hunk 1 above)
Comment (PLeVasseur) [reply subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md:2]: Good idea. Done.

**Pull request merged**